tools/topology/topology2/cavs/tplg.dot

```diff
@@ -0,0 +1,32 @@
+digraph topology {
+node [color=Red,fontname=Courier]
+edge [color=Blue, style=dashed]
+"copier.host.5.1"->"copier.HDA.6.1"
+
+"copier.host.7.1"->"copier.HDA.8.1"
+
+"copier.host.9.1"->"copier.HDA.10.1"
+
+"gain.2.1"->"copier.HDA.2.1"
+
+"mixin.1.1"->"mixout.2.1"
+
+"copier.HDA.4.1"->"gain.4.1"
+
+"mixin.4.1"->"mixout.3.1"
+
+"copier.DMIC.11.1"->"copier.host.10.1"
+
+"copier.host.1.1"->"gain.1.1"
+
+"gain.1.1"->"mixin.1.1"
+
+"mixout.2.1"->"gain.2.1"
+
+"mixout.3.1"->"gain.3.1"
+
+"gain.3.1"->"copier.host.3.1"
+
+"gain.4.1"->"mixin.4.1"
+
+}
```

Comment:
was this intended to be part of this PR?

Reply:
no. will remove